sections/Chapter 5 Sprint 3/Sprint3.tex

```diff
@@ -1,21 +1,9 @@
 \chapter{Sprint 3}
+We have chosen to combine our descriptions of the two previous sprints into one sprint. Sprint 2 was supposed to be the sprint where the search engine was fixed, but ended up being swallowed by planning and bureaucracy. For this reason, it also took sprint 3 to fully implement the search engine. Because of our choice to combine the two sprints, any reference to sprint 3 should be interpreted as referring to the combined sprint.
```

Comment:
Maybe "swallowed by planning and collaboration efforts between the groups knox."

Kunne være rart at vide hvor langt sprint 3 så egentlig var.